src/Commands/CreateUserCommand.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Commands;
+
+use App\Exceptions\User\CreateUserErrorException;
+use App\Repository\User\UserManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class CreateUserCommand extends Command
+{
+    private const DEFAULT_PASSWORD = 'ToBeChanged';
+
+    private $userManager;
+
+    public function __construct(UserManagerInterface $userManager)
+    {
+        parent::__construct();
+
+        $this->userManager = $userManager;
+    }
+
+    public function configure(): void
+    {
+        $this
+            ->setName('app.users.create')
+            ->setDescription('Create a new user into the CEM app')
+            ->setHelp('This command allows you to create a new user into the Community Event Manager. 
+            The mandatory information are the email address and the username of the new customer.
+            The username must be all alphanumeric with `_`, `-`, `.`, without spaces and be at least 5 characters long.
+            The password for this user will be by default \''.self::DEFAULT_PASSWORD.'\'.')
+            ->addArgument('emailAddress', InputArgument::REQUIRED, 'Email address')
+            ->addArgument('username', InputArgument::REQUIRED, 'Username')
```

Comment:
Do we have/need usernames?

Reply:
Username is mandatory on the User object, but I can put it optional and if it's not set, take the local-part of the email as username

Reply:
No worries, leave it there, sounds good to me (easier to understand events/logs when it's a known username).